style.css

```diff
@@ -0,0 +1,41 @@
+.calculator{
```

Comment:
Overall nice work on the styles! It looks good on all but phone screens. Could you please add some media queries to make the width of the calculator and the calcField larger on smaller screen sizes?

Check this video out for more info https://www.youtube.com/watch?v=5xzaGSYd7jM&t=2s

and a reference for how to pick breakpoint sizes https://www.freecodecamp.org/news/the-100-correct-way-to-do-css-breakpoints-88d6a5ba1862/ (don't worry about all the stuff about Sass right now, just read the intro and first part about the sizes)